### PR TITLE
#879 Inverted option for STEPS style

### PR DESCRIPTION
**Pull request by zdavkeos**
New line style like STEPS but drawn the opposite way.
Screenshot 2023-11-01 at 7 16 40 PM

### CODE REVIEW

**Comment by zdavkeos** [comment]
@facontidavide & @mikkelti I am not crazy about the name "Inverted Steps", that's qwt's terminology. I would be very open to any suggestions.

**Comment by mikkelti** [comment]
@zdavkeos Inspired by matplotlib's terminology, you could call them `Steps (pre)` and `Steps (post)` or something like that.

File: plotjuggler_app/plotwidget_editor.ui [hunk 1]
```diff
@@ -344,7 +344,14 @@
...
               <property name="text">
-               <string>Steps</string>
+               <string>Steps (pre)</string>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QRadioButton" name="radioStepsInv">
+              <property name="text">
+               <string>Inverted (post)</string>
```
Comment (facontidavide) [inline plotjuggler_app/plotwidget_editor.ui:354]:
```suggestion
               <string>Step (post)</string>
```

**Review by facontidavide — CHANGES_REQUESTED** [review]

**Comment by zdavkeos** [comment]
Good catch @facontidavide , Thanks!

**Pull request merged**